open-metadata-resources/open-metadata-tutorials/beginners-guide/running-kubernetes.md

```diff
@@ -0,0 +1,68 @@
+<!-- SPDX-License-Identifier: CC-BY-4.0 -->
+<!-- Copyright Contributors to the ODPi Egeria project. -->
+
+# Running Egeria in a self-contained environment using Kubernetes
+
+
+With this approach, you do not need to download, install, configure and understand all of the underlying components
+that Egeria itself uses.
+
+## Kubernetes
+[Kubernetes](https://kubernetes.io/) is an open platform for managing complex, containerized workloads. 
+
+You can get access to a Kubernetes environment (cluster) through many major cloud providers. 
+
+You can also install Kubernetes locally:
+ * For Windows & MacOS the easiest way is to [download docker desktop](https://hub.docker.com/?overlay=onboarding). After install click on the Docker icon, go to 'Preferences' and under 'resources' ensure you have at least 6GB dedicated to Docker. Then click on 'Kubernetes' and check the checkbox to enable.
+ * For Linux there is no easy installer, but there are many options. These also exist for Windows, Mac & indeed Raspberry Pi. Search for topics like `okd`,`minishift`,`microk8s`, `k3s` . 
+
+
+The remainder of these instructions assume that you have a Kubernetes (k8s) cluster configured, and 'kubectl' available. 
+
+## Helm
+
+[Helm](https://helm.sh/) is a packaging tool that helps in deploying applications to Kubernetes & is required by Egeria.
+
+Install [Helm v3.1 or above](https://github.com/helm/helm/releases) & ensure it is in your path.
+
+
+## Running the environment
+
+1. Download the latest [Egeria release](https://github.com/odpi/egeria/releases) & unpack
+   
+1. Navigate to the `open-metadata-resources/open-metadata-deployment/charts` directory
+   
+1. First we need to add an additional repository & retrieve dependencies we use for Egeria:
+
+    ```bash
+    $ helm repo add bitnami https://charts.bitnami.com/bitnami
```

Comment:
It's been ages since I've done this, so I'm trying to remember if these first two commands are actually necessary, or if they are done automatically anyway by the `helm dep update ...` command?

Reply:
I missed that relaxation. I just tested and you are correct. That explicit addition is not required (though it is harmless). I'll remove. less steps. Thanks